matterhorn.cabal

```diff
@@ -17,6 +17,8 @@ tested-with:         GHC == 8.0.2, GHC == 8.2.2, GHC == 8.4.4, GHC == 8.6.5, GHC
 data-files:          syntax/*.xml
                      syntax/language.dtd
                      syntax/LICENSE
+                     emoji/*.txt
```

Comment:
Please list all files explicitly here rather than using a wildcard. I don't want this wildcard accidentally including things that get added to that directory that aren't intended to be installed.

Reply:
Done.